client/influxdb.go

```diff
@@ -7,20 +7,66 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"net"
 	"net/http"
 	"net/url"
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/influxdb/influxdb/influxql"
 	"github.com/influxdb/influxdb/tsdb"
 )
 
+const (
+	// DEFAULT_HOST is the default host used to connect to an InfluxDB instance
+	DEFAULT_HOST = "localhost"
+	// DEFAULT_PORT is the default port used to connect to an InfluxDB instance
+	DEFAULT_PORT = 8086
+	// DEFAULT_TIMEOUT is the default connection timeout used to connect to an InfluxDB instance
+	DEFAULT_TIMEOUT = 0
+)
+
```

Comment:
Can you change the names to camel case? Also, spaces between the definition and the next comment line would be nice. :)